kcqrs-core/src/main/kotlin/com/clouway/kcqrs/core/AuditAwareAggregateRepository.kt

```diff
@@ -0,0 +1,30 @@
+package com.clouway.kcqrs.core
+
+/**
+ * Repository is representing an Repository which operates with the AggregateRoot objects.
+ *
+ * @author Miroslav Genov (miroslav.genov@clouway.com)
+ */
+interface AuditAwareAggregateRepository {
```

Comment:
Name looks strange ? What is the reason for the change form AggregateRepository?

Reply:
The Identity information which is required for audit purposes is now moved one level up to the callstack as Configuration object was added as dependency to the AggregateRepository. 

I'm open for suggestions for better name.